package.json

```diff
@@ -13,7 +13,7 @@
         "url": "https://github.com/WSStudios/vscode-fracas.git"
     },
     "engines": {
-        "vscode": "^1.61.0"
```

Comment:
@thisisthedave VS Code wouldn't let me build unless I up'd the `vscode` version number, but I'm worried anyone using an older version of VS Code won't be able to install this new package. My VS Code is `1.67.0` - maybe there's a way to target an older version when building the package?

Reply:
I think you need to install an older version to build for an older version. I have been encouraging people to upgrade as new versions of the extension are released so that we all stay up to date. The latest version is 1.69.2, so I'll bet most designers have already upgraded past 67.